packages/ui-kit/src/components/markdown.tsx

```diff
@@ -136,6 +135,12 @@ const Li = styled.li`
   > ul {
     list-style-type: circle;
   }
+  > * + :not(span) {
```

Comment:
This style change will affect any span tag and it might create undesired side effects
What you could to is specify a className in the PlanTag component and then change the negative rule to target just that specific span 

e.g.
```
> * + :not(span.planTag) {
```

Reply:
Good idea, thank you :)